helloWorld/README.md

```diff
@@ -0,0 +1,311 @@
+[base]: ../../docs/glossary.md#base
```

Comment:
Is the whole folder helloWorld copied from kustomize? Can you change the README.md to best serve the use case here?